lang/lang_en_fr.txt

```diff
@@ -280,6 +325,7 @@
 #MSG_FSENSOR
 "Fil. sensor"
 "Capteur Fil."
+
 #
 "Filam. runouts"
 "Fins de filament"
```

Comment:
"Fin de filament"

Reply:
`Fin de filament` is "The end of the filament", `Fins de filament` is "Endings of filament". I believe that "Endings of filament" is a better representation of "Filament runouts.

Reply:
have 1 one end of filament or more?

Reply:
This message appears under `Fail stats`, and is a total count of the number of times it has run out of filament during a print (detected an end of filament). So, this message should be plural, as, over the course of the printer's life, it will detect multiple ends of filament.

Reply:
Not sure what to change here. as it is limited to c=14 (just updated after checking code)
It is now `Fins filament`